core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java

```diff
@@ -528,6 +528,10 @@ void createMeasurementEvent(
                             offset = buffer.addTimestamp(offset, entity.getLongValue());
                             break;
 
+                        case ColumnType.DATE:
+                            offset = buffer.addDate(offset, entity.getLongValue() / 1000);
```

Comment:
Instead of allowing ILP timestamp to DATE column conversion the client could send ILP integer containing millis.
In your case that would mean that the `/1000` moves into `Table2IlpCopier` and no gain but clients calculating the timestamp value from strings like `2022-12-22` could start sending timestamps without realising that there is a more efficient way.
Maybe we should make it obvious on the API of the clients that millis integers should be sent for DATE columns by adding a `Sender::dateColumn(name, millis)` method.
That would definitely remove the confusion.

Reply:
We send both DATE and TIMESTAMP as PgWrire Timestamp so they are indistinguishable when you work with PgWire data cursor. I have to query the table metadata from the destination to find out what is the type.

I think giving a way to send time values in unified way is a better option, same as we do with integers.

Reply:
ok, let's add ILP timestamp to DATE conversion.
i see how this can make life easier in a number of situations.

maybe we should still add that `Sender::dateColumn(name, millis)` method to get users to use the more efficient way to ingest dates.
will create an issue for it.